mitosheet/src/plugin.tsx

```diff
@@ -161,7 +164,16 @@ function activateMitosheetExtension(
 
             const codeCell = getCellAtIndex(cells, mitosheetCallIndex + 1);
 
+            // We're removing the first line of the old code and the cell code because
+            // the cell code contains the analysis id and the old code does not
+            const oldCodeWithoutFirstLine = oldCode?.slice(1).join('\n');
+            const cellCodeWithoutFirstLine = getCellText(codeCell)?.split('\n').slice(1).join('\n');
+
```

Comment:
Given that this logic requires the same documentation in two places, I feel like we should factor it into a helper function with a clear and descriptive name.

Reply:
`src/jupyter/code.tsx` is probably a good file